testsuite/tests/kuadrant/authorino/conditions/section_conditions/test_response_condition.py

```diff
@@ -31,10 +31,10 @@ def test_skip_response(client, auth):
     assert response.status_code == 200
 
     # verify that response was not returned on a GET request
-    with pytest.raises(KeyError, match="Auth-Json"):
-        extract_from_response(response)
+    assert "simple" not in response.json()["headers"]
```

Comment:
Found a bug. This string should be `Simple` because HTTP headers capitalize first letter of header name. As this PR is already merged, can you create a new small one with this fix @pehala ?